tests/basic/execution.rs

```diff
@@ -1,15 +1,17 @@
 use shuttle::scheduler::RandomScheduler;
-use shuttle::{check, thread, Config, MaxSteps, Runner};
+use shuttle::sync::atomic::AtomicUsize;
+use shuttle::sync::Mutex;
+use shuttle::{check, check_random, thread, Config, MaxSteps, Runner};
 use std::panic::{catch_unwind, AssertUnwindSafe};
 use test_env_log::test;
-// Not actually trying to explore interleavings involving AtomicUsize, just using to smuggle a
-// mutable counter across threads
-use std::sync::atomic::{AtomicUsize, Ordering};
+// We use `std::sync::atomic::AtomicUsize` to smuggle a mutable counter across threads
+// without causing interleavings.
+use std::sync::atomic::{AtomicUsize as StdAtomicUsize, Ordering};
```

Comment:
Instead of the awkward renaming here, I would just `use shuttle::sync::atomic::AtomicUsize` inside the tests where you want it.

Reply:
I don't find it that awkward. My thinking was to make it easy to see in the tests that a non-Shuttle atomic is used, in particular since a test I added uses both the Shuttle and the std `AtomicUsize`. But I'm fine with reverting this, since the test will probably change to only use the Shuttle atomic.

Reply:
If you were going to use both in the same test, you could also just fully qualify the single use of `shuttle::sync::atomic::AtomicUsize::new(0)`.